recipes/cpp-subprocess/all/conanfile.py

```diff
@@ -0,0 +1,28 @@
+from conans import ConanFile, tools
+
+
+class CppSubprocess(ConanFile):
+    name = "cpp-subprocess"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/nextsilicon/cpp-subprocess#readme"
+    topics = ("subprocess", "os", "fork")
+    description = ("Subprocessing with modern C++, "
+                   "The only goal was to develop something that is as close as"
+                   "python subprocess module in dealing with processes.")
+    # No settings/options are necessary, this is header only
+    exports_sources = "subprocess.hpp"
```

Comment:
```suggestion
```

Reply:
Can you please explain (I'm new to conan packaging).

I believe I took it from a conan headers only example.